test/avro_ocf_tests.erl

```diff
@@ -59,6 +68,19 @@ write_file_test() ->
   {_Header, Type, Objs} = avro_ocf:decode_file(OcfFile),
   ?assertEqual([[{<<"f1">>, 1}, {<<"f2">>, <<"foo">>}]], Objs).
 
+write_deflate_file_test() ->
+  OcfFile = filename:join([priv_dir(), "deflate.ocf.test"]),
+  Store = undefined, %% should not require lookup
+  Fields = [ avro_record:define_field("f1", int, [])
+           , avro_record:define_field("f2", string, [])
+           ],
+  Type = avro_record:type("rec", Fields, [{namespace, "defalate.ocf.test"}]),
+  Obj = [{"f1", 1}, {"f2", "foo"}],
+  Meta = [{<<"avro.codec">>, <<"deflate">>}],
+  ok = avro_ocf:write_file(OcfFile, Store, Type, [Obj], Meta),
+  {_Header, Type, Objs} = avro_ocf:decode_file(OcfFile),
+  ?assertEqual([[{<<"f1">>, 1}, {<<"f2">>, <<"foo">>}]], Objs).
+
 root_level_union_test() ->
   OcfFile = filename:join([priv_dir(), "union.ocf.test"]),
   Store = undefined, %% should not require lookup
```

Comment:
We have 100% line coverage for `erlavro`, please make sure all lines are covered by tests.
currently below lines are not covered:
```
================ avro_ocf ================
182: is_invalid_codec_meta(<<"avro.codec">>, <<"null">>) -> false;
184: is_invalid_codec_meta(<<"avro.codec">>, _) -> true;
```

check the lines by:
```
make test
./scripts/cover-print-not-covered-lines.escript .eunit/cover.coverdata
```